### ISSUE TITLE
BUG: Items subtotal not displaying on visiting cart, on increasing/decreasing items subtotal is being displayed but not visible due to color contrast

### ISSUE THREAD

**Issue by AvinashShrivastav** [Anushkabh/krishiconnect#99]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

When items are added to the cart, the subtotal does not appear if you view the cart. 
<img width="716" alt="image" src="https://github.com/Anushkabh/krishiconnect/assets/110047194/bd13c630-ee2c-4d32-95c6-37d4d782473f">
However, when you increase or decrease the quantity of items, the subtotal then becomes visible, but it is displayed in black text on a black background.
<img width="475" alt="image" src="https://github.com/Anushkabh/krishiconnect/assets/110047194/2e717826-371b-43f8-b5fb-8e09ade8947f">

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue
